src/rapids_singlecell/preprocessing/_harmonypy_gpu.py

```diff
@@ -1,20 +1,3 @@
-# harmonypy - A data alignment algorithm.
-# Copyright (C) 2018  Ilya Korsunsky
-#               2019  Kamil Slowikowski <kslowikowski@gmail.com>
-#               2022  Severin Dicks <severin.dicks@gmail.com>
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
you can‘t do that. if harmonypy is distributed under the GPL, you can‘t just remove that.

you’d have to ask all authors if they’re OK with relicensing the code.